src/video/resources/liveStreams.js

```diff
@@ -226,6 +226,104 @@ class LiveStreams extends Base {
       `${buildBasePath(liveStreamId)}/playback-ids/${playbackId}`
     );
   }
+
+  /**
+   * Create a simulcast target
+   * @param {string} liveStreamId - The ID for the live stream
+   * @param {Object} params - Simulcast Target JSON parameters (e.g url and stream_key)
+   * @returns {Promise} - Returns a resolved Promise with a response from the Mux API
+   *
+   * @example
+   * const { Video } = new Mux(accessToken, secret);
+   *
+   * // Create a live stream playback ID
+   * Video.LiveStreams.createSimulcastTarget(liveStreamId, {url: 'rtmp://live.example.com/app', stream_key: 'difvbfgi', passthrough: 'Example Live Streaming service'});
+   *
+   * @see https://docs.mux.com/reference#create-a-simulcast-target
+   */
+  createSimulcastTarget(liveStreamId, params) {
+    if (!liveStreamId) {
+      return Promise.reject(
+        new Error('A Live Stream ID is required to create a simulcast target')
+      );
+    }
+
+    if (!params) {
+      return Promise.reject(
+        new Error(
+          'A url and a stream_key are required to create a simulcast target'
+        )
+      );
+    }
```

Comment:
Let's check to make sure these are required- our official docs might just not be indicating that they are.